pom.xml

```diff
@@ -183,16 +183,4 @@
         </dependencies>
     </dependencyManagement>
 
-    <build>
```

Comment:
These are breaking changes but the project has not been moved on to 3.0.0-SNAPSHOT.